test/unit/PackingGrid.spec.ts

```diff
@@ -106,4 +106,76 @@ describe("test PackingGrid", () => {
     expect(maxContentPos).to.be.closeTo(1200, 0.000001);
     expect(container!.style.height).to.be.equals("1200px");
   });
+  it(`should check if inlineSize:ContentSize is 1:2`, async () => {
```

Comment:
Can you add some conditions for the test names?
In this case, you should add in which condition inlineSize and contentSize should be 1:2